src/styles/components/_navbar.scss

```diff
@@ -106,6 +111,39 @@ html[data-theme='dark'] {
         }
       }
 
+      .icon-link {
+        &-mask {
+          &::before {
+            content: '';
+            width: 20px;
```

Comment:
I changed this to `19px` in the dev tools and it reproduces the icon sizing closer to production. 

The challenge that we have is in production the sizes are:
- Twitter icon: 19px
- Discord icon: 20px
- Github icon: 17px

So applying a one size fits all to them causes slight stretching compared to production.